### ISSUE TITLE
_freud.cpp and CMakeLists.txt

### ISSUE THREAD

**Issue by bdice** [glotzerlab/freud#90]
**[Original report](https://bitbucket.org/glotzer/freud/issue/90) by Matthew Spellings (Bitbucket: [mspells](https://bitbucket.org/mspells), GitHub: [klarh](https://github.com/klarh)).**

----------------------------------------

We should probably explicitly have _freud.cpp be the only thing we tell CMake to build inside freud/CMakeLists.txt instead of globbing. With globbing, it is possible that someone deletes _freud.cpp, successfully builds and installs freud, and gets errors when importing the module.

**Comment by bdice** [comment]
**Original comment by Eric Harper (Bitbucket: [harperic](https://bitbucket.org/harperic), GitHub: [harperic](https://github.com/harperic)).**

----------------------------------------

So, we would end up listing every py file in the file() command, but keep the custom target?

**Comment by bdice** [comment]
**Original comment by Matthew Spellings (Bitbucket: [mspells](https://bitbucket.org/mspells), GitHub: [klarh](https://github.com/klarh)).**

----------------------------------------

I guess in particular, in cpp/CMakeLists.txt,

```
file(GLOB_RECURSE PYX_CPPS ${CMAKE_CURRENT_SOURCE_DIR}/../freud/*.cpp ${CMAKE_CURRENT_SOURCE_DIR}/../freud/*.c) 
```

should just set the value to be "${CMAKE_CURRENT_SOURCE_DIR}/../freud/_freud.cpp"

**Comment by bdice** [comment]
**Original comment by Eric Harper (Bitbucket: [harperic](https://bitbucket.org/harperic), GitHub: [harperic](https://github.com/harperic)).**

----------------------------------------

That makes sense, and probably explains a weird commit Jens had a few weeks back. I'll implement.

**Comment by bdice** [comment]
**Original comment by Eric Harper (Bitbucket: [harperic](https://bitbucket.org/harperic), GitHub: [harperic](https://github.com/harperic)).**

----------------------------------------

Resolves Issue#90

**Comment by bdice** [comment]
**Original comment by Eric Harper (Bitbucket: [harperic](https://bitbucket.org/harperic), GitHub: [harperic](https://github.com/harperic)).**

----------------------------------------

Resolves Issue#90
